index.html

```diff
@@ -196,5 +196,47 @@ <h2>About me</h2>
             </ul>
         </div>
     </section>
+
+    <footer>
+        <div class="icons">
+        <p class="attract">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            Full name
+            Email address
+            Enter text here
+            Get in touch
+        </p>
```

Comment:
### Your Heading
![par](https://user-images.githubusercontent.com/93039359/186456905-2d5a9270-0f52-4932-95b8-2a9e46756644.png)

### Figma Heading
![paragraph-figma](https://user-images.githubusercontent.com/93039359/186456944-1cdea0b2-991b-4f6a-966b-3dfe9e18a4ba.png)

- [ ] Please use `h2` tag instead of `p` tag because it is the heading of your page :+1:
- [ ] Also, make it look similiar to the Figma template :blush: